dist/tools/picotool/.gitignore

```
@@ -0,0 +1 @@
+picotool
```

```suggestion
picotool
source/
```

The source folder has to be ignored as well and the `picosdk` folder is still missing a `.gitignore` that ignores the `picosdk/` folder.